recipes/tesseract/all/patches/0001-install-bundle.patch

```diff
@@ -0,0 +1,11 @@
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -537,7 +537,7 @@ get_target_property(tesseract_VERSION libtesseract VERSION)
+ get_target_property(tesseract_OUTPUT_NAME libtesseract OUTPUT_NAME)
+ configure_file(tesseract.pc.cmake ${CMAKE_CURRENT_BINARY_DIR}/tesseract.pc @ONLY)
+ install(FILES ${CMAKE_CURRENT_BINARY_DIR}/tesseract.pc DESTINATION lib/pkgconfig)
+-install(TARGETS tesseract RUNTIME DESTINATION bin LIBRARY DESTINATION lib ARCHIVE DESTINATION lib)
++install(TARGETS tesseract DESTINATION bin)
```

Comment:
so, aren't `RUNTIME`, `LIBRARY` and `ARCHIVE` destination needed to be set?
if so, why?
I think it's important to have libraries installed to the proper destination

Reply:
tesseract target is an executable. The lib is installed in another command.

Reply:
if it's an executable, it needs at least `RUNTIME DESTINATION`, right? (and maybe `BUNDLE DESTINATION` for OSX)

Reply:
DESTINATION alone set the destination for all types.

Reply:
And from CMake documentation: https://cmake.org/cmake/help/latest/command/install.html

> For each of these arguments given, the arguments following them only apply to the target or file type specified in the argument. If none is given, the installation properties apply to all target types. If only one is given then only targets of that type will be installed (which can be used to install just a DLL or just an import library.)